### ISSUE TITLE
Generalize and add a verifier for having different sizes of the typographic and atom components.

### ISSUE THREAD

**Issue by kyr0** [kyr0/defuss-shadcn#55]
`data-size` should be implemented with all atom components.
The following variants must always be supported:
- sm
- md (default, if not set)
- lg
